src/__experimental__/components/date/date.spec.js

```diff
@@ -227,6 +235,16 @@ describe('Date', () => {
     it('should update the input element to reflect the passed date', () => {
       expect(wrapper.update().find('input').prop('value')).toBe(getFormattedDate(mockDate));
     });
+
+    it('should return focus to the date input and the picker should not open', () => {
+      const instance = wrapper.find(BaseDateInput).instance();
+      expect(instance.inputFocusedViaPicker).toEqual(true);
+      const spy = spyOn(instance, 'updateSelectedDate');
+      const stateSpy = spyOn(instance, 'setState');
```

Comment:
I think that you shouldn't spy on the instance of the component

Reply:
Yeah I'll rewrite this as it's pretty much all implementation specific